dev/drivers/scripts/prep/glwu/jevs_glwu_wave_grid2obs_prep.sh

```diff
@@ -0,0 +1,74 @@
+#PBS -N jevs_glwu_wave_grid2obs_prep
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=00:15:00
+#PBS -l select=1:ncpus=1:mem=5GB
+#PBS -l debug=true
+
+
+set -x
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+
+export MODELNAME=glwu
+export NET=evs
+export COMPONENT=glwu
+export STEP=prep
+export RUN=wave
+export VERIF_CASE=grid2obs
+
+############################################################
+## read version file and set model_ver
+#############################################################
+
+versionfile=$HOMEevs/versions/run.ver
+. $versionfile
+export model_ver=$glwu_ver
+
+#############################################################
+# Load modules
+#############################################################
+
+module reset
+module load prod_envir/${prod_envir_ver}
+source $HOMEevs/dev/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh
+
+evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
+
+############################################################
+# set some variables
+#############################################################
+
+export envir=prod
+export SENDCOM=${SENDCOM:-YES}
+export SENDECF=${SENDECF:-YES}
+export SENDDBN=${SENDDBN:-NO}
+export KEEPDATA=${KEEPDATA:-YES}
+export SENDMAIL=${SENDMAIL:-YES}
+
+export MAILTO='alicia.bentley@noaa.gov,samira.ardani@noaa.gov'
```

Comment:
While I wouldn't mind getting the GLWU emails, GLWU is actually a regional model and the `MAILTO` addresses should actually be Samira and Andrew!

@SamiraArdani-NOAA @malloryprow @AndrewBenjamin-NOAA

Reply:
@AliciaBentley-NOAA, I can replace your email address with Andrew's.